src/store/annotations/reducer.ts

```diff
@@ -3,7 +3,11 @@ import { AnnotationsState } from './types';
 import { createAnnotationAction, fetchAnnotationsAction, setActiveAnnotationIdAction } from './actions';
 
 const activeAnnotationId = createReducer<AnnotationsState['activeId']>(null, builder =>
-    builder.addCase(setActiveAnnotationIdAction, (state, { payload: annotationId }) => annotationId),
+    builder
+        .addCase(setActiveAnnotationIdAction, (state, { payload: annotationId }) => annotationId)
+        /* Set the active annotation back to null, Preview will set the active ID with a an event which will trigger
```

Comment:
Probably don't need this portion, as it's evident in the code:
> Set the active annotation back to null

Reply:
Can we follow up on this flow when we re-fine setting the activeAnnotationId, shouldn't be in the scope of this PR, but I want this to work when we use it.

Reply:
I'm saying that the comment itself has redundant information in it, basically.